Library/MapleBacon/MapleBacon/Storage/NSUUID+V5.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  NSUUID+V5.swift
+//  vault
+//
+//  Created by Danilo Topalovic on 14.02.16.
+//  Copyright (c)  2016 nerdsee. All rights reserved.
+//
+
+import Foundation
+import CommonCrypto
+
+// MARK: - V5 UUID
+extension NSUUID {
+    
+    /**
+     Convenience init for building a uuid from V5 Standard using a
+     UUID namespace and a string "name"
+     
+     - parameter ns:   the namespace UUID
+     - parameter name: the string
+     */
+    convenience init(namespace ns: UUID, name: String) {
+        
+        var uuidBytes: UInt8 = 0
+        (ns as NSUUID).getBytes(&uuidBytes)
+        
+        let namespaceData: Data = NSData(bytes: &uuidBytes, length: 16) as Data
+        let nameData: Data      = NSData(bytes: name.cString(using: String.Encoding.utf8), length: name.characters.count) as Data
```

Comment:
formatting for one (aligning the = sign) and shorter `name.cString(using: .utf8)`